### ISSUE TITLE
Check capitalized

### ISSUE THREAD

**Issue by LEv145** [terrencepreilly/darglint#191]
Example of a function:
```py
def foo(iter, index: int) -> int:
    """
    get page by key

    Args:
        iter: iterator
        index: start index

    Returns:
        len of iterator at index
    """
 ```
 
 It would be convenient if the linter caused warnings:
 ```py
def foo(iter, index: int) -> int:
    """
    get page by key  # Darglint: First line should end with a period
                     # Darglint: First line should be properly capitalized

    Args:
        iter: iterator  # Darglint: Args line should end with a period
                        # Darglint: Args line should be properly capitalized

        index: start index # Darglint: Args line should end with a period
                           # Darglint: Args line should be properly capitalized

    Returns:
        len of iterator at index  # Darglint: Returns line should end with a period
                                  # Darglint: Returns line should be properly capitalized
    """
 ```
I have not found such functionality
 
This maintains a uniform style, how here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

It would be possible to make an option for this

**Comment by terrencepreilly** [comment]
See the [scope](https://google.github.io/styleguide/pyguide.html#381-docstrings) section of the readme.  I think you're looking for a tool like pydocstyle.

**Comment by LEv145** [comment]
Pycodestyle does not take this into account with google style, it would be useful to add this feature

You can make it optional, how mypy with `strict` mode
